src/System.Security.Cryptography.Csp/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netcoreapp-Windows_NT;
+      netstandard-Unix;
+      netcoreapp-Unix;
```

Comment:
Do we need both of these for Unix?  (I don't know one way or the other, but I don't think we have any netcoreapp-specific API here, so I don't know why the tests aren't just netstandard.  Or netcore.  or something.)